pytorch/training/docker/2.0/py3/Dockerfile.cpu

```diff
@@ -257,10 +250,11 @@ ARG PT_TORCHVISION_URL
 ARG PT_TORCHAUDIO_URL
 ARG PT_TORCHDATA_URL
 
-# Install AWS-PyTorch and other torch packages
-RUN pip uninstall -y torch torchvision torchaudio torchdata \
- && pip install --no-cache-dir -U ${PT_SM_TRAINING_URL} ${PT_TORCHVISION_URL} ${PT_TORCHAUDIO_URL} ${PT_TORCHDATA_URL}
-
+# force remove to prevent removing all packages depending on pytorch
+RUN /opt/conda/bin/conda remove -y --force pytorch \
+  && /opt/conda/bin/mamba install -y pytorch=${PYTORCH_VERSION} \
```

Comment:
do we need `cpuonly` flag here?

Also do we need to install pytorch twice as I see we are doing it twice in this file

Reply:
although it might not be needed, for added the cpuonly package for guarantee. 
And yes we need to install the ec2 package first and then the sm package